src/main/java/frc/robot/utilities/pid/PIDFConfigurable.java

```diff
@@ -7,6 +7,21 @@
     @Override
     PIDFCoefs getCoefs();
 
+    default void setCoefs(PIDFCoefs coefs) {
+        getCoefs().set(coefs);
+    }
+
+    PIDFCoefs getRemoteCoefs();
+
+    default void setRemoteCoefs(PIDFCoefs coefs) {
+        getRemoteCoefs().set(coefs);
+    }
```

Comment:
@ overide

Reply:
It's not overriding because the types are different

Reply:
@Mboesn resolve?